docs/en/ingest-management/troubleshooting.asciidoc

```diff
@@ -4,12 +4,113 @@
 
 This release of Ingest Management is an experimental release. With this early release we explicitly invite you to install and test it **in a test environment**, but it is absolutely possible that you run into problems and need to tweak your setup to get it running.
 
-We have collected the most common known problems below. If your question isn't answered here, please contact us https://ela.st/ingest-manager-feedback{here}. Your feedback is very valuable to us.
+We have collected the most common known problems below. If your question isn't answered here, please review open issues in the https://github.com/elastic/kibana/issues, https://github.com/elastic/beats/issuess, https://github.com/elastic/package-registry/issues, and contact us https://ela.st/ingest-manager-feedback{here}. Your feedback is very valuable to us.
 
+* <<ingest-manager-not-in-cloud-kibana>>
+* <<ingest-management-setup-fails>>
+* <<ingest-manager-app-crashes>>
+* <<agent-enrollment-timeout>>
+* <<enrolled-agent-not-showing-up>>
+* <<where-are-the-agent-logs>>
+* <<what-is-my-agent-config>>
+* <<where-is-the-data-agent-is-sending>>
+* <<i-deleted-my-agent>>
+* <<i-need-to-stop-agent>>
+* <<i-rebooted-my-host>>
+* <<what-is-the-endpoint-package>>
+* <<ingest-manager-not-in-on-prem-kibana>>
+
+[[ingest-manager-not-in-cloud-kibana]]
+=== The {ingest-manager} app is not listed in my Kibana cloud navigation list.
```

Comment:
Do you want all of this info to be on the same page, or each problem to be a separate topic?

Reply:
The same page look a good idea, I assume we that page to reduce in the upcoming releases.